docs/Server-Methods.md

```diff
@@ -143,6 +143,10 @@ The logger instance, check [here](https://github.com/fastify/fastify/blob/master
 #### inject
 Fake http injection (for testing purposes) [here](https://github.com/fastify/fastify/blob/master/docs/Testing.md#inject).
 
+<a name="add-schema"></a>
+#### addSchema
+`fastify.addSchema(schemaObj)`, adds a shared schema to the Fastify instance, then you can reuse it everywhere in your application just by writing the schema id that you need. You can see [here](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#shared-schema) how it works.
```

Comment:
"`fastify.addSechema(schemaObj)` adds a shared schema to the Fastify instance. This allows you to reuse it ..."

Reply:
I didn't see that "here" link before. That sentence could be:

"To learn more, see [shared schema example] in the [Validation and Serialization] documentation."

http://uxmovement.com/content/why-your-links-should-never-say-click-here/